supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/api/request/GetRequestHandler.scala

```diff
@@ -125,11 +125,11 @@ trait GetRequestHandler { this: RequestHandler =>
             case None => Success(executionEvents, machineType, zone, instanceName)
           }
         } else if (isQuotaDelayed(events)) {
-          AwaitingCloudQuota
+          AwaitingCloudQuota(executionEvents)
         } else if (operation.hasStarted) {
-          Running
+          Running(executionEvents)
         } else {
-          Initializing
+          Initializing(executionEvents)
```

Comment:
Clarification: these statuses-with-events hold a list of all the events? or just the events that caused the job to be in that status?

Reply:
All of the events.